gettsim/tests/test_arbeitslosengeld_ui.py

```diff
@@ -27,17 +28,22 @@
 ]
 OUT_COL = "m_alg1"
 years = [2010, 2011, 2015, 2019]
-tax_policy_data = load_tax_benefit_data()
+
+
+@pytest.fixture
+def input_data():
+    file_name = "test_dfs_ui.csv"
+    out = pd.read_csv(ROOT_DIR / "tests" / "test_data" / file_name)
+    return out
 
 
 @pytest.mark.parametrize("year", years)
-def test_ui(year):
-    file_name = "test_dfs_ui.ods"
-    df = load_test_data(year, file_name, input_cols, pd_kwargs={"true_values": "TRUE"})
+def test_ui(input_data, tax_policy_data, year):
+    year_data = input_data[input_data["year"] == year]
```

Comment:
Chaining commands is more readable. ``df = input_data.loc[input_data.year.eq(year), input_cols].copy()``

Reply:
But is `input_data.year` really nicer than `input_data["year"]`. Philipp told me the other way around.

Reply:
My argument was mainly about the chaining. Feel free to adopt whatever style you like to pick a column. I am in favor of the dot, because it seems cleaner to me. I think Janos is in favor of your approach because it better distinguishes between columns and methods.

Reply:
indexes, no dots please. Pandas explicitly discourages using dot syntax (basically they just added it so you get tab completion in interactive mode). Makes it difficult to change things, too, although that might be less of a concern for 'year'

Reply:
Quote Philipp: The dot is an remaining from the stata users. i have no idea if thats true :)

Reply:
I didn't chain because I use the year data again later to get the expected results. Then I would need to condition a second time on year.

Reply:
No idea where you may have dot syntax in Stata...

Reply:
I will shed this bad habit from now on :).

Reply:
> because it better distinguishes between columns and methods

I fully agree